src/util/Injected/Utils.js

```diff
@@ -135,11 +135,16 @@ exports.LoadUtils = () => {
         } else if (options.parseVCards && typeof (content) === 'string' && content.startsWith('BEGIN:VCARD')) {
             delete options.parseVCards;
             try {
-                const parsed = window.Store.VCard.parseVcard(content);
+                // eslint-disable-next-line no-useless-escape
+                const rgx_vcard = new RegExp('^(?:\\s*BEGIN:VCARD\\s*\\n\\s*VERSION:[2-9]\.[0-9](?=\\n))'
```

Comment:
We have `parseVcard` in the module `WAWebVcardParsingUtils`, please use it instead.

Reply:
It was not working .  parseVcard function does not seem to be present.

You can try sending a VCard string via SendMessage to test:

let vcfmessage = 'BEGIN:VCARD\n' +
  'VERSION:4.0\n' +
  'FN:Tom Smith\n' +
  'N:Smith;Tom;;;\n' +
  'TEL;type=CELL;type=VOICE;waid=11234567890:+1 123 456 7890\n' +
  'END:VCARD';

client.sendMessage(number, vcfmessage);

The original library does not work.